docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -176,325 +115,451 @@
     "\n",
-    "Use the Sampler mode of Fire Opal Performance Management to run a Bernstein–Vazirani circuit. This algorithm, used to find a hidden string from the outputs of a black box function, is a common benchmarking algorithm because there is a single correct answer."
+    "| Name  | Type  | Description | Required | Default  | Example |\n",
+    "|------|------|----------|----------|-----------|-----------|\n",
+    "| pubs       | `EstimatorPubLike` or `list[EstimatorPubLike]` | One or more tuples containing the inputs listed under `EstimatorPubLike` components   | Yes   | N/A   | `(circuit, observables, parameter_values)`   |\n",
+    "| instance   | `str`   | The hub/group/project to use in that format  | No  | A Premium access instance is randomly chosen if your account has access to multiple instances | `\"hub1/group1/project1\"`     |\n",
+    "| backend_name| `str`  | The name of the backend | No  |  The least busy backend that your instance has access to | `\"ibm_fez\"` |\n",
+    "| options| `dict`  | Input options; see Options section for more details | No  |  See the Options section for details | `{\"default_shots\": 2048}` |\n",
+    "\n",
+    "**`EstimatorPubLike` components:**\n",
+    "- A single circuit defined as a `QuantumCircuit` or in OpenQASM 2.0 or 3.0 string format.\n",
+    "- One or more observables that specify the expectation values to estimate, in any of the formats denoted in the list \"Supported observables formats\".\n",
+    "- (Optional) A collection of parameter values to bind the circuit against, which follow the same [array broadcasting rules](./primitive-input-output#broadcasting-rules) as the `QiskitRuntime` primitives.\n",
+    "- (Optional) A target precision for expectation values to estimate.\n",
+    "- (Optional) A dictionary of run options, such as the default shot count.\n",
+    "\n",
+    "**Supported observables formats:**\n",
+    "- Any one of the `ObservablesArrayLike` formats, such as `Pauli`, `SparsePauliOp`, `PauliList`, or `str`\n",
+    "- A Pauli string: `\"XY\"`\n",
+    "- A dictionary - Pauli strings with coefficients: `{\"XY\": 0.5, \"YZ\": 0.3}`\n",
+    "- A list of Pauli strings: `[\"XY\", \"YZ\", \"ZX\"]`\n",
+    "- A list of Pauli strings with coefficients: `[(\"XY\", 0.5), (\"YZ\", 0.3)]`\n",
+    "- A nested list of Pauli strings: `[[\"XY\", \"YZ\"], [\"ZX\", \"XX\"]]`\n",
+    "- A nested list of Pauli strings with coefficients: `[[(\"XY\", 0.1), (\"YZ\", 0.2)], [(\"ZX\", 0.3), (\"XX\", 0.4)]]`\n",
+    "\n",
+    "**Supported backends:**\n",
+    "The following list of backends are currently supported. If your device is not listed, [reach out to Q-CTRL](https://form.typeform.com/to/iuujEAEI?typeform-source=q-ctrl.com) to add support.\n",
```

Comment:
Thank you for adding this!

Reply:
Note that any time one of these devices retires, this list will need to be updated, so we'll need to keep an eye on it.